.circleci/config.yml

```diff
@@ -30,64 +33,90 @@ commands:
           key: android=tools-v1-{{ checksum "scripts/install-android-tools.sh" }}-{{ arch }}
           paths:
             - ~/android-tools
-
-  create-launch-android-emulator:
-    description: "create and launch android emulators"
-    steps:
-      - run:
-          name: create AVD
-          command: |
-            echo "no" | avdmanager --verbose create avd --force \
-              --name "Pixel_3a_API_26" \
-              --package "system-images;android-26;google_apis_playstore;x86"
-
-      - run:
-          name: start AVD
-          command: emulator @Pixel_3a_API_26 -no-window -no-audio
-          background: true
-
-      - run:
-          name: wait for emulator
-          command: |
-            adb wait-for-device shell 'while [[ -z $(getprop dev.bootcomplete) ]]; do sleep 1; done;'
```

Comment:
Removed in favor of [CircleCI Android orb](https://circleci.com/developer/orbs/orb/circleci/android)